src/env/commands/emufs.S

```diff
@@ -138,8 +138,16 @@ MFSini:
 	pushw %cs
 	popw %ds
 
-	movw	$0x3000, %ax
+	movw	$0x3099, %ax	# Trigger for MOS version (PC-MOS only)
```

Comment:
Can we, by any chance, get the redirector
version from fatfs? As it usually knows
what DOS we boot. Or maybe only ask
fatfs whether we are MOS? Then emufs.sys
can be spared from the MOS-specific code.

Reply:
Sounds good, thanks.